assets/l10n/app_en.arb

```diff
@@ -665,9 +665,13 @@
   "@unknownUserName": {
     "description": "Name placeholder to use for a user when we don't know their name."
   },
+  "youLabel": "(you)",
+   "@youLabel": {
+     "description": "Label shown after the current user's display name in direct messages."
+      },
```

Comment:
Bump https://github.com/zulip/zulip-flutter/pull/2019#discussion_r2607832074 : please look carefully at how we do indentation in this file